Doc/Tutorial/chapter_testing.tex

```diff
@@ -3,7 +3,7 @@ \chapter{The Biopython testing framework}
 
 Biopython has a regression testing framework (the file
 \verb|run_tests.py|) based on
-\href{http://docs.python.org/library/unittest.html}{unittest},
+\href{https://docs.python.org/2/library/unittest.html}{unittest},
```

Comment:
Please use https://docs.python.org/3/library/unittest.html rather than the obsolete Python 2 link.

Reply:
I use the reference to Python 3 documentation.